### PR TITLE
create bugbash md

### PR DESCRIPTION
**Pull request by botany-dummy**
Here is PR summary to have reviewed

### CODE REVIEW

File: bugbash.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Bug Bash File for activity
+who goes first
```
Comment (botany-demo) [inline bugbash.md]: Each word in headings should start with a cap `# Bug Bash File For Activity`

Comment (botany-dummy) [reply bugbash.md]: Thank you

**Comment by botany-dummy** [comment]
also updated to add `?` at the end of sentence.

File: bugbash.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Bug Bash File For Activity
+who goes first?
```
Comment (botany-demo) [inline bugbash.md]: what do you think about making it `Who` instead of `who`

Comment (botany-dummy) [reply bugbash.md]: good catch

**Pull request merged**